README.md

```diff
@@ -89,7 +89,32 @@ the filecoindot offchain worker has been set up!
 
 #### 3. set the the offchain worker's account as relayer
 
-_TODO_
+In the provided [substrate-node-example](./substrate-node-example/README.md), we configured the [ManagerOrigin](https://github.com/ChainSafe/filecoindot/blob/5da77ea5ad329ce1e9009f5afd4f54dabbdaa6ee/substrate-node-example/runtime/src/lib.rs#L278) of `filecoindot` with `frame_system::EnsureRoot<AccountId>`, so here we need to `add_relayer` with sudo access.
```

Comment:
the hardcoded link will be broken eventually, I think just using `type ManagerOrigin` should be fine, without the link